Gemfile

```diff
@@ -53,7 +53,6 @@ gem 'devise-async'
 gem 'newrelic_rpm', '~> 3.14'
 gem 'whenever', require: false
 gem 'pg_search'
-gem 'acts_as_list'
```

Comment:
@andreslucena Nope! We thought it's not useful right now because we are just using the order by position and not manipulating lists.